src/main/java/org/littil/api/contact/service/ContactMapper.java

```diff
@@ -0,0 +1,21 @@
+package org.littil.api.contact.service;
+
+import org.littil.api.contact.api.ContactPostResource;
+import org.littil.api.contact.repository.ContactEntity;
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(componentModel = "cdi")
+public interface ContactMapper {
+
+    Contact toDomain(ContactPostResource contactResource);
+
+    @Mapping(source = "createdDate", target = "contactDate")
```

Comment:
apart veld toevoegen voor contactDate (Geen dubbele betekenis toekkennen aan auditable velden) ?

Reply:
ja, beetje dezelfde als met sender; for-now is dat ook createdBy